frontend/src/features/properties/map/research/ResearchTabsContainer.tsx

```diff
@@ -8,9 +8,16 @@ import { Api_ResearchFile } from 'models/api/ResearchFile';
 import React, { useState } from 'react';
 
 import ResearchSummaryView from './detail/ResearchSummaryView';
+import { FormKeys } from './FormKeys';
 
 export interface IResearchTabsContainerProps {
   researchFile: Api_ResearchFile;
+  // The "edit key" identifies which form is currently being edited: e.g.
+  //  - property details info,
+  //  - research summary,
+  //  - research property info
+  //  - 'none' means no form is being edited.
+  setEditKey: (editKey: FormKeys) => void;
   setEditMode: (isEditing: boolean) => void;
```

Comment:
I had to add one more indicator/flag to make sure the appropriate update sub-form was shown given the various "edit" buttons for different tabs/sections